test/datalog_tests/rust_api_test/src/main.rs

```diff
@@ -43,7 +52,15 @@ fn main() -> Result<(), String> {
     //   of output relations.  Should only be used for debugging in order to dump
     //   the contents of output tables using `HDDlog::dump_table()`.  Otherwise,
     //   indexes are the preferred way to achieve this.
-    let (hddlog, init_state) = HDDlog::run(1, false)?;
+    let (hddlog, init_state) = HDDlog::new(
```

Comment:
We should generate a convenience function for this.  There's no way we can expose this API to users.